src/GlobalDependencyMap.js

```diff
@@ -165,18 +161,14 @@ class GlobalDependencyMap {
 	}
 
 	getTemplatesThatConsumeCollections(collectionNames) {
-		let templates = new Set();
-		for (let name of collectionNames) {
-			let collectionName = GlobalDependencyMap.getCollectionKeyForEntry(name);
-			if (!this.map.hasNode(collectionName)) {
-				continue;
-			}
-			for (let node of this.map.dependantsOf(collectionName)) {
-				if (!node.startsWith(GlobalDependencyMap.COLLECTION_PREFIX)) {
-					templates.add(node);
-				}
-			}
-		}
+		let templates = new Set(
```

Comment:
I’m not sure if this style of chainability is more readable than the code on the left hand side, specifically when it has this many links in the chain!